### ISSUE TITLE
`validate-objects-count` complains when no objects are found

### ISSUE THREAD

**Issue by yegor256** [objectionary/lints#538]
This is my XMIR:

```
<object/>
```

It doesn't have any `<o/>` elements (due to a broken syntax). Then, I'm getting this:

```
<error check="validate-objects-count" line="0" severity="critical">Only one object per source file is allowed and required</error>
```

This message is misleading. The `validate-objects-count` lint should ignore situations when no `<o/>` elements are present. Maybe other lints should complain about such a case (we can create them), but not this one.

**Comment by h1alexbel** [comment]
@yegor256 in lints, we don't have `validate-objects-count` lint. This check comes from EO parser: `org/eolang/parser/parse/validate-objects-count.xsl`
